src/github.com/getlantern/golog/golog.go

```diff
@@ -79,7 +80,17 @@ func LoggerFor(prefix string) Logger {
 	l := &logger{
 		prefix: prefix + ": ",
 	}
-	l.traceOn, _ = strconv.ParseBool(os.Getenv("TRACE"))
+	trace := os.Getenv("TRACE")
```

Comment:
I made a change here so instead of print all traces we can trace  selected packages
`TRACE=flashlight, enproxy, detour ...`